_posts/2024-06-01-Announcing-Inbound-IPv6-support.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Announcing inbound IPv6 support in public preview"
+author_name: "Mads Damgård"
+toc: true
+toc_sticky: true
+---
+
+I am happy to announce the first part of our IPv6 implementation in App Service. Public preview of inbound IPv6 support for multi-tenant apps on Premium SKUs, Functions Consumption, Functions Elastic Premium, and Logic Apps Standard). We'll be adding IPv6 support in four stages.
+
+1. IPv6 inbound support (multi-tenant)
+1. IPv6 non-vnet outbound support (multi-tenant)
+1. IPv6 vnet outbound support (multi-tenant and App Service Environment v3)
+1. IPv6 vnet inbound support (App Service Environment v3 - both internal and external)
+
+Limitations in public preview:
+
+* Only a subset of regions are supported - see the list below.
+* Basic and Standard tier is currently not supported.
+* Functions Consumption may temporarily have extra IP addresses in the DNS result.
+* Functions Consumption and Elastic Premium may not remove the IPv4 address in IPv6 mode.
```

Comment:
Didn't we determine this was just a DNS caching thing and it goes away?

Reply:
I believe this is referring to the issue where old function apps that were created using a secondary flex consumption A record will not have their A record removed in IPv6 mode